resources/views/customer/deposit/mail.blade.php

```diff
@@ -0,0 +1,15 @@
+@component('mail::message')
+Good day,
+
+This is your deposit receipt.
+
+{{-- Table --}}
+@component('mail::table')
+| Status          | Ticket Date | Total Amount | Account Name |
+| ------------- |:-----------:| -----------:| -----------:|
+| {{ $deposits['status'] }} | {{ $deposits['deposit_ticket_date'] }} | {{ $deposits['total_amount'] }} | {{ $deposits['account_name'] }} |
```

Comment:
Can you make this into two columns? Something Like this
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |

```
| Name        | Value         |
|:-----------|:------------|
| Status        | test status |
| Ticket Date | 2022-08-16 |
```